Symfony/CS/Tests/Fixer/Contrib/ProtectedToPrivateFixerTest.php

```diff
@@ -48,47 +48,47 @@ private static function f6(){}
         $attributesAndMethodsFixed = str_replace('protected', 'private', $attributesAndMethodsOriginal);
 
         return array(
-            'final-extends' => array(
```

Comment:
ah yes, force of habbit (i.e. custom fixer I've ran), I'll revert those changes.